connectors/services/sync.py

```diff
@@ -35,13 +37,13 @@ class SyncService(BaseService):
     def __init__(self, config):
         super().__init__(config)
         self.idling = self.service_config["idling"]
-        self.hb = self.service_config["heartbeat"]
+        self.heartbeat_interval = self.service_config["heartbeat"]
```

Comment:
We could think about renaming the option in the yaml file also. Would be more consistent and also obvious what's the consequence of setting heartbeat to something

Reply:
Yes, but probably in a separate PR given that we're backporting this change to 8.7.

Can address in 8.8